src/CacheException.php

```diff
@@ -2,9 +2,16 @@
 
 namespace Psr\SimpleCache;
 
+/**
+ * PHP 5 compatibility for \Throwable interface.
+ */
+if (version_compare(PHP_VERSION, '7.0.0', 'lt')) {
```

Comment:
if this ever gets accepted, you should use `PHP_VERSION_ID` numeric compare and check that somebody else has not created `\Throwable` class or class alias.